src/TextUI/Option/Colors.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * @since Class available since Release 6.0.0
+ */
+final class PHPUnit_TextUI_Option_Colors extends PHPUnit_TextUI_Option_Option
```

Comment:
@sebastianbergmann What are your point of view with the `final` keyword?

fyi, [Here](http://verraes.net/2014/05/final-classes-in-php/) is why I put all my class final in my template.